crates/turborepo-lib/src/run/summary/mod.rs

```diff
@@ -407,22 +406,28 @@ impl<'a> RunSummary<'a> {
         spaces_client_handle: SpacesClientHandle,
         ended_at: DateTime<Local>,
         exit_code: i32,
-    ) -> Result<(), Error> {
-        let spinner = turborepo_ui::start_spinner("...sending run summary...");
+    ) {
+        let spinner_fut = async {
+            tokio::time::sleep(std::time::Duration::from_millis(1000)).await;
+            turborepo_ui::start_spinner("...sending run summary...")
+        };
 
-        spaces_client_handle.finish_run(exit_code, ended_at).await?;
+        // We log the error here but don't fail because
+        // failing to send the space shouldn't fail the run.
+       let (result, spinner) = tokio::join!(spaces_client_handle.finish_run(exit_code, ended_at), spinner_fut);
```

Comment:
Actually, wouldn't this wait on both the spinner and `finish_run`? So we'd be waiting a whole second unnecessarily. @chris-olszewski